### PR TITLE
add price section in elephpant page

### LINKED ISSUE #29: Elephpant page - Add price and buy now link
- Add section with elephpant price
- Add button/link "buy now", redirect for sympla

### PR DESCRIPTION
**Pull request by alessandrofeitoza**
This resolve #29

### CODE REVIEW

File: 2019/en/elephpant.html [hunk 1]
```diff
@@ -228,8 +228,63 @@ <h4 class="info-title">Low-price ticket</h4>
...
+
+			<div class="col-md-7 ml-auto">
+				<div class="tab-content text-center" >
+					<div class="tab-pane active" id="personal" role="tabpanel">
+						<div class="space-top"></div>
+						<div class="row">
+							<div class="col-md-6">
+								<div class="card card-pricing" data-color="green">
+									<div class="card-body">
+										<h6 class="card-category text-primary">Just ElePHPant</h6>
```
Comment (tdutrion) [inline 2019/en/elephpant.html]:
```suggestion
										<h6 class="card-category text-primary">ElePHPant only</h6>
```

File: 2019/en/elephpant.html [hunk 2]
```diff
@@ -228,8 +228,63 @@ <h4 class="info-title">Low-price ticket</h4>
...
+										<h1 class="card-title">R$ 100,00</h1>
+										<ul>
+											<li><b>1</b> ElePHPant</li>
```
Comment (tdutrion) [inline 2019/en/elephpant.html]:
```suggestion
											<li><b>1</b> ElePHPant plush toy</li>
```

File: 2019/en/elephpant.html [hunk 3]
```diff
@@ -228,8 +228,63 @@ <h4 class="info-title">Low-price ticket</h4>
...
+											<li><b>Receive</b> at the event venue</li>
```
Comment (tdutrion) [inline 2019/en/elephpant.html]:
```suggestion
											<li><b>Collect</b> at the venue on the conference days</li>
```

File: 2019/en/elephpant.html [hunk 4]
```diff
@@ -228,8 +228,63 @@ <h4 class="info-title">Low-price ticket</h4>
...
+											<li><b>Someone</b> else can get you</li>
```
Comment (tdutrion) [inline 2019/en/elephpant.html]:
```suggestion
											<li><b>Delegate</b> the ElePHPant collection if you can't attend</li>
```

File: 2019/en/elephpant.html [hunk 5]
```diff
@@ -228,8 +228,63 @@ <h4 class="info-title">Low-price ticket</h4>
...
+											<li><b>Does not</b> give access to the event</li>
+										</ul>
+										<a href="https://www.sympla.com.br/darkmira-tour-php-2019__420712" class="btn btn-neutral btn-round">Adopt an elePHPant</a>
```
Comment (tdutrion) [inline 2019/en/elephpant.html]:
```suggestion
										<a href="https://www.sympla.com.br/darkmira-tour-php-2019__420712" class="btn btn-neutral btn-round">Adopt your ElePHPant</a>
```

**Review by tdutrion — DISMISSED** [review]

File: 2019/en/elephpant.html (hunk 5 above)
Comment (tdutrion) [reply 2019/en/elephpant.html]: @alessandrofeitoza I've added some comments regarding the English version.

**Review by cgrandval — APPROVED** [review]

**Pull request merged**